org.mobsya.ThymioSuite.metadata.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not a blocker, but this should be upstreamed.

Reply:
I can do that if necessary (we own both repo) - but i figured it would be simpler this way for reviews

Reply:
We prefer if it's maintained upstream as it's not a Flatpak/Flathub specific thing.